packages/data-core/vite.config.ts

```diff
@@ -1,13 +1,20 @@
+/// <reference types="vitest" />
 import { defineConfig } from "vite"
 import dts from "vite-plugin-dts"
 
 // https://vitejs.dev/config/
 export default defineConfig({
   build: {
     lib: {
-      entry: "src/data.ts",
+      entry: "src/data-core.ts",
       formats: ["es", "cjs"],
     },
+    rollupOptions: {
+      external: ["source-jsonapi"],
```

Comment:
I will figure out a solution so we don't have to do this.

Reply:
updated all to use `external: [/node_modules/],`